arm-outputs/arm-outputsV2/task.json

```diff
@@ -81,6 +81,20 @@
             "label": "Filter deployment name",
             "required": false,
             "helpMarkDown": "Optional string to filter deployments by. This can be useful if you have concurrent deployments to the same resource group. Deployment names in Azure Pipelines are the name of the JSON file plus the date and time, so a file `CreateKeyVault.json` could have a deployment name of `CreateKeyVault-20180025-151538-0688`. In this case, if you want to filter to deployments of this file enter `CreateKeyVault` as the filter.<br/><br/>This uses [micromatch](https://github.com/micromatch/micromatch) under the hood (ignoring case) so us wildcards like *"
+        },
+        {
+            "name": "environment",
```

Comment:
The environment should be pulled automatically from the ARM service connection provided by the user. That should be accomplished via the line in index.ts
`var environmentName = tl.getEndpointDataParameter(connectedServiceNameARM, "environment", false);`

Is environment as task input a change that will be used for another purpose?

Reply:
That's awesome, I didn't know that! Will fix this!